packages/frontend-shared/src/components/Switch.vue

```diff
@@ -22,7 +22,7 @@
 withDefaults(defineProps<{
   value: boolean
   size?: 'sm' | 'md' | 'lg' | 'xl'
-  name: string // required for an id so that an external <label> can be associated with the switch
+  labelId: string // required for an id so that an external <label> can be associated with the switch
```

Comment:
The Switch component was built to be used like:

```html
<Switch name="setting" .... />
<label for="setting">Enabled Setting</label>
```

Everywhere this was used was flagged with the violation `buttons must have discernible text`. This seems like it _should_ be valid given it aligns with the suggested markup [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/switch_role#adding_the_switch_role_in_aria).

According to https://github.com/dequelabs/axe-core/issues/3696, this structure is flagged as a violation as it does not play nice with JAWS/IE 11 screen readers and axe supports those browsers.

Reply:
Thanks for the context. I've actually asked the axe-core team about surfacing the "accessibility supported" reasoning behind certain failing techniques in axe-core itself as part of the failure, and my understanding is that this would be a pretty big project and not something they plan to take on.